ingest/sqs/router/usecase/optimized_routes_test.go

```diff
@@ -551,169 +552,130 @@ func (s *RouterTestSuite) TestValidateAndFilterRoutes() {
 	}
 }
 
-// Validates that USDT for UMEE quote does not cause an error.
-// This pair originally caused an error due to the lack of filtering that was
-// added later.
-func (s *RouterTestSuite) TestGetBestSplitRoutesQuote_Mainnet_USDTUMEE() {
```

Comment:
Note: all the removed tests in this file were converted to table-driven to minimize the number of places we need to change each time.

The name of the new test is `TestGetOptimalQuote`